cmd/generate.go

```diff
@@ -25,31 +27,43 @@ func NewGenerateCommand() *cobra.Command {
 			return nil
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
+			var manifests []unstructured.Unstructured
+			var errs []error
+			var err error
+
 			path := args[0]
-			files, err := listYamlFiles(path)
-			if len(files) < 1 {
-				return fmt.Errorf("no YAML files were found in %s", path)
-			}
-			if err != nil {
-				return err
+			if path == StdIn {
+				manifests, err = readManifestData(cmd.InOrStdin())
+				if err != nil {
+					errs = append(errs, err)
+				}
+			} else {
+				files, err := listYamlFiles(path)
+				if len(files) < 1 {
+					return fmt.Errorf("no YAML files were found in %s", path)
+				}
+				if err != nil {
+					return err
+				}
+
+				manifests, errs = readFilesAsManifests(files)
 			}
 
-			manifests, errs := readFilesAsManifests(files)
 			if len(errs) != 0 {
 				// TODO: handle multiple errors nicely
 				return fmt.Errorf("could not read YAML files: %s", errs)
 			}
 
 			v := viper.New()
-			config, err := config.New(v, &config.Options{
+			cmdConfig, err := config.New(v, &config.Options{
```

Comment:
I update this because there could be a potential naming conflict with the package name above